library/CMakeLists.txt

```diff
@@ -17,7 +17,6 @@ else()
     find_package(nlohmann_json QUIET REQUIRED CONFIG)
     find_package(
         Protobuf
-        ${protobuf_version}
```

Comment:
This would break my workflow - it would allow CMake to find system-installed or Chromium-installed protobuf.

The most important thing is of course that your libprotobuf version matches your protoc version exactly. 

Should I assume you're trying not to use conan? 

Would it be helpful if we had the version to be passed into the CMake part of the build (which is typically run by conan, but shouldn't have to be) as a variable like `cmake -D PROTOBUF_VERSION=5.27.0` ?

Reply:
i'm using conan. removing the version check might have been unnecessary, the blocker for me was the fix below on line 76

Reply:
and prob because im using the dev version of protobuf 
```
protoc --version
libprotoc 31.0-dev
```

Reply:
Thanks for pointing it out. I think "libprotobuf" is just more correct regardless, but I'll check to make sure it works for me.

Is it OK if I leave this PR open here for a while? You can use your fork for now and after I get my stuff together I'll let you know and we'll see if things work OK for you... sound good?

Reply:
yeah let's leave it open as things shake out